client/core/core.go

```diff
@@ -3637,7 +3660,7 @@ func (c *Core) Withdraw(pw []byte, assetID uint32, value uint64, address string)
 	if err != nil {
 		return nil, err
 	}
-	const feeSuggestion = 100
```

Comment:
Nasty surprise paying $10 to withdraw when 50 cents is sufficient.

Reply:
Oof. I don't know why tf I would have done this.